shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -498,6 +503,11 @@ - (void)performAction:(FlutterTextInputAction)action withClient:(int)client {
   return _shell->Screenshot(type, base64Encode);
 }
 
+#pragma mark - FlutterBinaryMessengerContainer
```

Comment:
This may have been refactored but FlutterBinaryMessengerContainer doesn't seem to refer to anything concrete.

Reply:
Done, removed here and in the ViewController.